src/openzaak/components/catalogi/admin/forms.py

```diff
@@ -51,6 +51,35 @@ def _make_required(self, field: str):
             return
         self.fields[field].widget.required = True
 
+    def has_changed(self):
+        # Since Django thinks the below fields are always changed we
```

Comment:
I think this is the best way to do this.  This was always evaluating to `True` because the `ArrayField`s and `PositiveIntegerField` send their data as a string and since it's saved differently in the model `has_changed` was always returning True.  (Maybe worth filing a bug in Django?).

Since this would mean always sending a notification when a ZaakType was saved I'm manually checking these fields to see if they've actually changed.

Reply:
sounds okay, that setup with `initial-` hidden form fields recently bit me too, there's something odd going on with it.